jennchenn/uwexplore
# Add prof, tags, prereqs, antireqs to course object

## Implementation description

- Add prof, tags, prereqs, antireqs to course object
- tags, prereqs, antireqs are just lists of strings



## Steps to test

1. Fetch courses:
```
curl localhost:5000/courses?code=1
```
3. Verify return contains instructor, tags, prereq, antireq
```
{
    "antirequisites": [
      "MUSIC 140"
    ],
    "ceab_design": 0.0,
    "ceab_eng": 0.0,
    "ceab_math": 0.0,
    "ceab_sci": 0.0,
    "code": "140",
    "course_type": "LIST C",
    "cse_weight": 36.0,
    "department": "MUSIC",
    "description": "An examination of the styles, forms, and development of 20th century popular music. The social, commercial, and technological aspects of popular music are considered.",
    "id": "640246f5f414e1e1ebd4cf35",
    "name": "Popular Music and Culture",
    "prerequisites": [],
    "sections": [
      {
        "capacity": 200,
        "class_number": 1509,
        "day": [
          "MONDAY"
        ],
        "end_date": "Sun, 17 Dec 2023 00:00:00 GMT",
        "end_time": 1250.0,
        "enrolled_number": 200,
        "id": "640246f2f414e1e1ebd4cf29",
        "instructor": "Virgil Van Cleef",
        "location": "EV3 3201",
        "number": "001",
        "start_date": "Thu, 07 Sep 2023 00:00:00 GMT",
        "start_time": 1140.0,
        "term_code": "1229",
        "type": "LEC"
      }
    ],
    "tags": [
      "music",
      "culture"
    ]
  }
```



## What should reviewers focus on?

- does it make sense
